.editorconfig

```diff
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
yes of course

Question: do you want to apply this fix on other python scripts (for example, the python code from the documentation) ?